docs/index.md

```diff
@@ -24,7 +24,7 @@ source, visit [Building Spark](building-spark.html).
 
 Spark runs on both Windows and UNIX-like systems (e.g. Linux, Mac OS). It's easy to run
 locally on one machine --- all you need is to have `java` installed on your system `PATH`,
-or the `JAVA_HOME` environment variable pointing to a Java installation.
+or the `JAVA_HOME` environment variable pointing to an existing Java installation.
```

Comment:
What's the ambiguity here?

Reply:
A very subtle change here. Trying to make this more "noobie friendly" by implying that user needs to set up JRE before running spark.

Reply:
I din't think this line is confusing ..